tests/testthat/test-local.R

```diff
@@ -1,10 +1,28 @@
-test_that("expected options are set by default", {
+test_that("local context is 'as promised' inside test_that()", {
+  # high-level expectations or expectations that preceded testthat 3e
   expect_true(is_testing())
   expect_equal(testing_package(), "testthat")
```

Comment:
This is *not* actually true if you run the test via `test_file()`:

```
> test_file("tests/testthat/test-local.R")

══ Testing test-local.R ═════════════════════════════════════════════════════════
[ FAIL 1 | WARN 0 | SKIP 0 | PASS 17 ]

── Failure (test-local.R:4:3): local context is 'as promised' inside test_that() ───────────────────────────
testing_package() (`actual`) not equal to "testthat" (`expected`).

`actual`:   ""        
`expected`: "testthat"

[ FAIL 1 | WARN 0 | SKIP 0 | PASS 17 ]
```

Should this be fixed?